java-checks/src/main/java/org/sonar/java/checks/AbstractPrintfChecker.java

```diff
@@ -77,45 +83,22 @@
   protected static final MethodMatchers JAVA_UTIL_LOGGER_LOG_MATCHER = MethodMatchers.or(
```

Comment:
All constants related to logger can be `private` and moved in the PrintfMisuseCheck class